core/src/test/resources/filter.properties

```diff
@@ -0,0 +1,2 @@
+keywords=TransactionImple,Periodic Recovery,TransactionSynchronizationRegistryImple
+
```

Comment:
How did you decide on this list of keywords. If they are controlling which log lines are passed on for further processing then there are many others that need to be included, for example BasicAction.